modules/programs/broot.nix

```diff
@@ -0,0 +1,256 @@
+          > $out
+      '';
+
+  brootConf = {
+    verbs = cfg.verbs;
+    skin = cfg.skin;
+  };
+
+in
+
+{
+  meta.maintainers = [ maintainers.aheaume ];
+
+  options.programs.broot = {
+    enable = mkEnableOption ''
+      A better way to navigate directories.
+    '';
+
+    enableBashIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Bash integration.
+      '';
+    };
+
+    enableZshIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Zsh integration.
+      '';
+    };
+
+    enableFishIntegration = mkOption {
+      default = true;
+      type = types.bool;
+      description = ''
+        Whether to enable Fish integration.
+      '';
+    };
+
+    verbs = mkOption {
+      type = types.listOf types.attrs;
+      default = [
+        {
+          invocation = "p";
+          execution = ":parent";
+        }
+        {
+          invocation = "edit";
+          shortcut = "e";
+          execution = "$EDITOR {file}";
+        }
+        {
+          invocation = "create {subpath}";
+          execution = "$EDITOR {directory}/{subpath}";
+        }
+        {
+          invocation = "view";
+          execution = "less {file}";
+        }
+      ];
+      example = [
+        {
+          invocation = "p";
+          execution = ":parent";
+        }
+        {
+          invocation = "edit";
+          shortcut = "e";
+          execution = "$EDITOR {file}";
+        }
+        {
+          invocation = "create {subpath}";
+          execution = "$EDITOR {directory}/{subpath}";
+        }
+        {
+          invocation = "view";
+          execution = "less {file}";
+        }
+      ];
+      description = ''
+        Define new verbs.
+
+        The possible attributes are:
+
+        - invocation (mandatory): how the verb is called by the user, with placeholders for arguments
+        - execution (mandatory): how the verb is executed
+        - shortcut (optional): an alternate way to call the verb (without the arguments part)
+        - leave_broot (optional): whether to quit broot on execution (default: true)
+        - from_shell (optional): whether the verb must be executed from the parent shell (needs br, default: false)
+      '';
+    };
+
+    skin = mkOption {
+      type = types.attrs;
+      default = {};
+      example = {
+        status_normal_fg = "grayscale(18)";
+        status_normal_bg = "grayscale(3)";
+        status_error_fg = "red";
+        status_error_bg = "yellow";
+        tree_fg = "red";
+        selected_line_bg = "grayscale(7)";
+        permissions_fg = "grayscale(12)";
+        size_bar_full_bg = "red";
+        size_bar_void_bg = "black";
+        directory_fg = "lightyellow";
+        input_fg = "cyan";
+        flag_value_fg = "lightyellow";
+        table_border_fg = "red";
+        code_fg = "lightyellow";
+      };
+      description = ''
```

Comment:
Again will have to use docbook markup, otherwise this will all end up being a single unbroken line.